css/style.css

```
@@ -49,80 +61,81 @@ img {
   height: auto;
 }
 
-.visually-hidden {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  margin: -1px;
-  border: 0;
-  padding: 0;
-  clip: rect(0 0 0 0);
-  overflow: hidden;
-  }
+.page {
+  height: 100%;
+}
+
+.page-body {
+  min-height: 100%;
+  display: grid;
+  grid-template-rows: min-content 1fr min-content;
+  align-content: start;
+
+}
 
  /*****       index        *****/ 
 
 /* Main navigation */
 
-.welcome {
-  text-align: center;
-  margin: auto;
-  padding-top: 77.35px;
-}
-
 .main-navigation {
   background-color: var(--basic-white);
-  background-color: crimson;
-  margin: 0;
+  margin: auto;
   color: var(--basic-black);
-  height: 70px;
-  min-height: 70px; 
-}
-
-.main-navigation-wrapper {
+  width: 1200px;
   display: flex;
   flex-direction: column;
   align-items: center;
   position: relative;
-  flex-wrap: wrap;
+  flex-wrap: wrap; 
 }
 
 .logo {
-  background-color: blueviolet;
   order: -1;
   position: absolute;
+  align-self: center;
+  z-index: 2;
 }
 
-.site-navigation  {
-  background-color: blue;
-  min-height: 70px;
-  width: 100%;
-  margin: 0;
+.navigation {
+  list-style: none;  
+}
+
+ul.site-navigation {
   display: flex;
   flex-wrap: wrap;
   align-items: center;
-  position: relative;
+  width: 1200px;
```

лучше не привязываться к ширине, если она поменяется для макета, то нужно будет много где править, 100% гибче
```suggestion
  width: 100%;
```